### PR TITLE
Chore/libweb 1127 monorepo refactoring pt. 1

### PR DESCRIPTION
**Pull request by silesky**
In order to reduce risk and make reviewing easier, I'm refactoring in stages. The goal is to have _only_ the runtime-agnostic code live in core. 😸 

This creates a "node" and "browser" directory and moves the appropriate tests into each. This also moves analytics into "core". Eventually, we will be able to remove runtime specific logic from core.

New Structure (note, "packages" is not introduced yet). See [SDD](https://paper.dropbox.com/doc/SDD-Break-up-analytics-next-into-discrete-packages--BiRfAKKbwIYRfHR2lXRqXjMdAg-kJZSy7qZEFOaBgYCtUDIm) for Summary!

image

### CODE REVIEW

File: .husky/pre-push [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn test
```
Comment (silesky) [inline .husky/pre-push:4]: Sneaking this in =) 

(I've found this is the least controversial of all the hooks as it runs infrequently and fairly quickly, and I've found it extremely helpful).

**Review by zikaari — APPROVED** [review]
Looking nice! Good start 👍

File: src/__tests__/standalone-analytics.test.ts [hunk 2]
```diff
@@ -1,247 +0,0 @@
-import jsdom, { JSDOM } from 'jsdom'
```
Comment (chrisradek) [inline src/__tests__/standalone-analytics.test.ts:1]: I don't see these tests tests got moved to - are these still covered?

**Review by chrisradek — APPROVED** [review]
Sweet, :ship::it:

**Pull request merged**